src/pages/workspace/accounting/PolicyAccountingPage.tsx

```diff
@@ -107,7 +106,6 @@ function PolicyAccountingPage({policy}: PolicyAccountingPageProps) {
                           shouldCallAfterModalHide: true,
                           disabled: isOffline,
                           iconRight: Expensicons.NewWindow,
-                          shouldShowRightIcon: connectedIntegration !== CONST.POLICY.CONNECTIONS.NAME.SAGE_INTACCT,
```

Comment:
I am not sure why this check was added earlier.

Reply:
Yeah, I think you're right in getting rid of it. Even for Intacct this navigates somewhere, so we want the right icon.

Reply:
Do you want me to set it `true` ?